### PR TITLE
Test mode upgrade

### PR DESCRIPTION
**Pull request by finnsherman**
test mode for IRI

### CODE REVIEW

File: src/main/java/frc/robot/commands/testing/TestCommand.java [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  public boolean isFinished() {
+    return !RobotState.isTest();
+  }
+
+  @Override
+  public void end(boolean interrupted) {
+    shooterSubsystem.stop();
+    turretSubsystem.stop();
+    intakeSubsystem.stop();
+    turretSubsystem.prepareToIntake();
```
Comment (agasser) [inline src/main/java/frc/robot/commands/testing/TestCommand.java]: Why `prepareToIntake()` at the end? Usually you would just want to stop the subsystems when you're done with them.

**Review by agasser — CHANGES_REQUESTED** [review]
Really close, just one question...

File: src/main/java/frc/robot/commands/testing/TestCommand.java (hunk 1 above)
Comment (finnsherman) [reply src/main/java/frc/robot/commands/testing/TestCommand.java]: No idea what I was thinking when I wrote that, but its fixed now.

File: src/main/java/frc/robot/commands/testing/TestCommand.java [hunk 2]
```diff
@@ -202,6 +202,8 @@ public void end(boolean interrupted) {
     shooterSubsystem.stop();
     turretSubsystem.stop();
     intakeSubsystem.stop();
-    turretSubsystem.prepareToIntake();
+    turretSubsystem.stopPitch();
+    turretSubsystem.stopYaw();
+    turretSubsystem.runRollers(RotationsPerSecond.zero());
```
Comment (agasser) [inline src/main/java/frc/robot/commands/testing/TestCommand.java]: Line 203 calls `turretSubsystem.stop()`, so I don't think you need lines 205-207, the turret will already be stopped.

(Side note, `runRollers(RotationsPerSecond.zero())` doesn't turn the rollers off, it actively drives and holds the velocity at zero. `stopRollers()` is the method to turn the roller motors off.)

**Review by agasser — CHANGES_REQUESTED** [review]

**Review by agasser — APPROVED** [review]
Test it on the robot, and then you're good to merge it!

**Pull request merged**